dilithium.py

```diff
@@ -507,7 +515,7 @@ def verify(self, pk_bytes, m, sig_bytes):
         w_prime = self._use_hint(h, Az_minus_ct1, 2*self.gamma_2)
         w_prime_bytes = w_prime.bit_pack_w(self.gamma_2)
         
-        return c_tilde == self._h(mu + w_prime_bytes, 32)
+        return c_tilde == self._h(mu + w_prime_bytes, self.ctildebytes)
         
 Dilithium2 = Dilithium(DEFAULT_PARAMETERS["dilithium2"])
 Dilithium3 = Dilithium(DEFAULT_PARAMETERS["dilithium3"])
```

Comment:
```suggestion
Dilithium3 = Dilithium(DEFAULT_PARAMETERS["dilithium3"])
ML_DSA_44 = Dilithium2
ML_DSA_65 = Dilithium3
ML_DSA_87 = Dilithium5
```
I would propose to add those lines for proper import references. Otherwise there might be some confussion when using this implementation in a project where people are not too familiar with (currently draft) FIPS-204.

Reply:
You're right, this is a good idea.